cmd/oci-runtime-tool/generate.go

```diff
@@ -92,21 +92,23 @@ var generateCommand = cli.Command{
 	Before: before,
 	Action: func(context *cli.Context) error {
 		// Start from the default template.
-		specgen := generate.New()
+		specgen, err := generate.New(nil)
```

Comment:
How about getting `--os` value before `generate.New()`?
Then we don't have to pass nil any more.